kerchunk/grib2.py

```diff
@@ -1,8 +1,14 @@
+from ast import Import
 import base64
 import logging
 
-import cfgrib
-import eccodes
+try:
+    import cfgrib
+    import eccodes
+except ModuleNotFoundError as err:
+    if err.name == 'cfgrib': raise ImportError('cfgrib is needed to kerchunk GRIB2 files. Please install it with `conda install -c conda-forge cfgrib`')    
```

Comment:
Let's just have the one error saying that you need both of these packages. It can `raise from` so that the original import error with the named package shows up, or include its name in an f-string.

Reply:
Got it, just checking for `cfgrib` now